docs/deployment.rst

```diff
@@ -27,6 +27,32 @@ The host or Docker image must be set up with a sandbox user, sandbox virtualenv,
 
 See `2U's Dockerfile <https://github.com/edx/public-dockerfiles/blob/main/dockerfiles/codejail-service.Dockerfile>`__ for an example of how you could set up your host or container. (As of March 2025, there is `not yet a Tutor plugin <https://github.com/openedx/codejail-service/issues/26>`__.)
 
+App user UID
+============
+
+The UID under which the webapp runs must be running as few other processes as possible, otherwise the ``NPROC`` setting will need to be elevated to the point of uselessness.
+
+Recommendation
+--------------
+
+* Select a UID and GID for the webapp user: ``python3 -c 'import random; print(random.randrange(3000, 2 ** 31))'``. Create a user and group for the webapp user using this as its UID and GID.
+* Do the same for the sandbox user, with a new value.
+* In Kubernetes (or other orchestration system), limit the number of codejail-service instances that are permitted to be scheduled on the same host as each other.
```

Comment:
Is there anything you can say about this limit? Limit to 1? Limit to X, where X should be less than Y? Other?

Reply:
Unfortunately, no. I think it's going to depend greatly on traffic load and other things specific to a deployer. (I don't even know what 2U should do in this regard, yet.)

Reply:
Maybe I should instead say "consider limiting" to make it clear that this isn't necessarily required?